core/editor.js

```diff
@@ -275,19 +276,19 @@
 
 			// If there is no other customConfig in the chain, fire the
 			// "configLoaded" event.
-			if ( CKEDITOR.getUrl( editor.config.customConfig ) == customConfig || !loadConfig( editor ) )
+			if ( CKEDITOR.getUrl( editor.config.customConfig ) == customConfig || !loadConfig( editor ) ) {
 				editor.fireOnce( 'customConfigLoaded' );
+			}
+
```

Comment:
I think we don't usually leave an empty line before the closing brackets, so there is no need for adding one here.

Reply:
I've added this line because at the first glance outer `else` looks like it belongs to this inner `if`.
![image](https://user-images.githubusercontent.com/10279586/88513133-074bec80-cfe8-11ea-9c91-9515981deb3f.png)
**vs**
![image](https://user-images.githubusercontent.com/10279586/88513159-116deb00-cfe8-11ea-8b61-712ba8149ec9.png)

Reply:
Ok, we can leave it this way :+1: